WordPress/WordPressStatsWidgets/LockScreenWidgets/Views/LockScreenSingleStatView.swift

```diff
@@ -0,0 +1,57 @@
+import SwiftUI
+import WidgetKit
+
+@available(iOS 16.0, *)
+struct LockScreenSingleStatView: View {
+    @Environment(\.widgetFamily) var family: WidgetFamily
+    let viewModel: LockScreenSingleStatViewModel
+
+    var body: some View {
+        if family == .accessoryRectangular {
+            ZStack {
+                AccessoryWidgetBackground().cornerRadius(8)
+                VStack(alignment: .leading) {
+                    Text(viewModel.siteName)
+                        .frame(maxWidth: .infinity, alignment: .leading)
+                        .font(.system(size: 11))
```

Comment:
🔤 If possible, we should try supporting [dynamic type](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/docs/dynamic-type-guidelines.md) so users who have the text size increased or decreased in the accessibility settings would have reflected that in our apps in our widgets. I think setting the fixed system size does not support dynamic type out of the box. Could you check if dynamic type support is possible for these lock screen widgets?

[You can check more of our guidelines in the docs](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/docs).

Reply:
I admit I intended to use fixed font because the space is not enough in the lock screen widget, and sorry for I didn't highlight this decision during the design discussion.

I just implicitly said the font size of the number is smaller than the original design due to space issues. 
That's why I promoted another design to get the ability to enlarge the font.
<img width="272" alt="image" src="https://user-images.githubusercontent.com/3096210/225554861-67c20ee9-557c-40b8-8588-4a5502762f5f.png">

For now, we don't have enough space to enlarge the font (only remain the edge padding spaces).
so my suggested solution is how about we change the layout to horizontal for the large font? (the right side one in the screenshot)

Reply:
Thanks for the explanation!

I'm checking my current lock widgets, and they are not reacting to dynamic type changes either. As you explained, there's very little space to deal with. Even if we move to a horizontal layout, we will get other types of issues.

Let's stick to the layout we've chosen. 👍